<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vic's Macros</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>256</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>151</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="bHotKey">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Hotkey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bPress">
       <property name="text">
        <string>Press</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bMultiPress">
       <property name="text">
        <string>Multi Press</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bHold">
       <property name="text">
        <string>Hold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bRelease">
       <property name="text">
        <string>Release</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bWrite">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bWait">
       <property name="text">
        <string>Wait</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>151</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="bClick">
       <property name="text">
        <string>Mouse Click</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bRightClick">
       <property name="text">
        <string>Right Click</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bDoubleClick">
       <property name="text">
        <string>Double Click</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bMouseMove">
       <property name="text">
        <string>Mouse Move (x,y)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bMoveRelative">
       <property name="text">
        <string>Mouse Move (x+x2, y+y2)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bClickandDrag">
       <property name="text">
        <string>Click and Drag</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bMousePosition">
       <property name="text">
        <string>Mouse Position</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>301</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="bCopy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bMoveUp">
       <property name="text">
        <string>Move Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bMoveDown">
       <property name="text">
        <string>Move Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bTrigger">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="Repeats">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>500</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bRepeats">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string># of Repeats</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInstructions"/>
    <addaction name="actionAcceptable_Inputs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="actionAcceptable_Inputs">
   <property name="text">
    <string>Acceptable Inputs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
